xds/src/test/java/io/grpc/xds/EdsLoadBalancerTest.java

```diff
@@ -519,7 +525,7 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
 
   @Test
   public void handleLocalityAssignmentUpdates_pickersUpdatedFromChildBalancer() {
-    deliverResolvedAddresses(new XdsConfig(null, null, "edsServiceName1", null));
+    deliverResolvedAddresses(new XdsConfig(CLUSTER_NAME, null, null, "edsServiceName1", null));
```

Comment:
Can you change  "edsServiceName1" in this test to null? The `clusterLoadAssignment` then is expected to use `CLUSTER_NAME`. That is what is happening in real case.

Reply:
Changed test cases to use null `edsServiceName`. (This doesn't matter much for things we are aiming to test, although being closer to real use case is better).